torch/ao/quantization/_dbr/quantization_state.py

```diff
@@ -117,6 +117,10 @@ def __init__(
         # to its final value after tracing.
         self.needs_dtype_transform_on_outputs = True
 
+        # For debugging only, stores the types of ops seen by the parent which
+        # did not require op hooks.
+        self.seen_op_types_without_op_hooks: Set[Callable] = set()
```

Comment:
so these ops are just added to the set and not connected to the quantizable subgraph right?

Reply:
correct, this is useful when onboarding a new model to check what ops are already supported and what are not.  Once there is parity between DBR and FX graph mode quant, we can delete this code.